strings/check_panagram.py

```diff
@@ -0,0 +1,24 @@
+# Created by sarathkaul on 12/11/19
+
+
+def check_panagram(input_str: str) -> None:
```

Comment:
Please remove all printing from our function so it is an algorithm that could be dropped into a larger program...
```python
def check_panagram(input_str: str="The quick brown fox jumps over the lazy dog") -> bool:
```
All that printing should happen outside the function (algorithm).

Please add a test:
```
>>> check_panagram("The quick brown fox jumps over the lazy dog")
False
```